### PR TITLE
OLS-157: Proper distribution of endpoints unit tests

### PR DESCRIPTION
**Pull request by onmete**
## Description

Just moving tests around to reflect the main package (unit tests should mirror that).

No code is changed here.

## Type of change

- [x] Refactor

### CODE REVIEW

File: tests/unit/app/endpoints/test_health.py
Comment (bparees) [inline tests/unit/app/endpoints/test_health.py:1]: `test_health.py` and not `test_healthy.py`

Comment (onmete) [reply tests/unit/app/endpoints/test_health.py:1]: It is `test_health.py`. I don't see `test_healthy.py` anywhere.

File: tests/unit/app/endpoints/test_feedback.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""Unit tests for feedback endpoints."""
```
Comment (tisnik) [inline tests/unit/app/endpoints/test_feedback.py:1]: it checks handlers, not the endpoint

File: tests/unit/app/endpoints/test_health.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-"""Unit tests for health REST API endpoints handlers."""
+"""Unit tests for health endpoints."""
```
Comment (tisnik) [inline tests/unit/app/endpoints/test_health.py:1]: dtto, it checks handlers, not endpoint

File: tests/unit/app/endpoints/test_ols.py [hunk 3]
```diff
@@ -51,3 +65,132 @@ def test_nothing_is_logged_when_provider_and_model_not_set(self, caplog):
...
+    llm_request = LLMRequest(query="Tell me about Kubernetes")
+    response = ols.conversation_request(llm_request)
+    assert (
+        response.response
+        == "Kubernetes is an open-source container-orchestration system..."
+    )
+    assert len(response.conversation_id) > 0
+
+    # valid question, yaml
+    mock_validate_question.return_value = [
```
Comment (tisnik) [inline tests/unit/app/endpoints/test_ols.py:100]: it will be removed soon, please delete too

**Review by tisnik — COMMENTED** [review]
commented

File: tests/unit/app/endpoints/test_feedback.py (hunk 1 above)
Comment (onmete) [reply tests/unit/app/endpoints/test_feedback.py:1]: right

File: tests/unit/app/endpoints/test_ols.py (hunk 3 above)
Comment (onmete) [reply tests/unit/app/endpoints/test_ols.py:100]: Should I remove it here now? Wouldn't it be better to remove tests in the same PR where relevant logic is removed?

**Review by tisnik — COMMENTED** [review]
looks ok

File: tests/unit/app/endpoints/test_health.py
Comment (bparees) [reply tests/unit/app/endpoints/test_health.py:1]: sorry, perhaps i hallucinated last night

**Pull request merged**